src/core/geometry/qgslinestring.cpp

```diff
@@ -1059,6 +1108,80 @@ void QgsLineString::points( QgsPointSequence &pts ) const
   }
 }
 
+void QgsLineString::setPoints( qint64 size, const double *x, const double *y, const double *z, const double *m )
```

Comment:
size_t would be a more natural type
```suggestion
void QgsLineString::setPoints( size_t size, const double *x, const double *y, const double *z, const double *m )
```

Reply:
done